build_resolvers/test/resolver_test.dart

```diff
@@ -367,8 +367,10 @@ int? get x => 1;
                 ''',
               }, (resolver) async {
                 var lib = await resolver.libraryFor(entryPoint);
-                expect(lib.languageVersionMajor, sdkLanguageVersion.major);
-                expect(lib.languageVersionMinor, sdkLanguageVersion.minor);
+                expect(lib.languageVersion.effective.major,
+                    sdkLanguageVersion.major);
```

Comment:
nit: I'm starting to add trailing commas when I force a wrap in an arg list – do what you like, thoug!

Reply:
we don't do that, generally